### PR TITLE
Add new symfony dotenv structure

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | -
| Related issues/PRs | -
| License | MIT
| Documentation PR | -

#### What's in this PR?

Add new symfony dotenv structure

#### Why?

The recipes where changed:

https://github.com/symfony/recipes/blob/master/symfony/framework-bundle/3.3/config/bootstrap.php
https://github.com/symfony/recipes/blob/master/symfony/framework-bundle/3.3/public/index.php
https://github.com/symfony/recipes/blob/master/symfony/console/3.3/bin/console

Using the 3.3 recipes to be compatible between 3.4 and 4.0

### CODE REVIEW

File: bin/console.php [hunk 1]
```diff
@@ -4,35 +4,35 @@
...
-if (!isset($_SERVER['APP_ENV'])) {
-    if (!class_exists(Dotenv::class)) {
-        throw new \RuntimeException('APP_ENV environment variable is not defined. You need to define environment variables for configuration or add "symfony/dotenv" as a Composer dependency to load variables from a .env file.');
-    }
-    (new Dotenv())->load(__DIR__.'/../.env');
+$input = new ArgvInput();
+if (null !== $_ENV['APP_ENV'] = $input->getParameterOption(['--env', '-e'], null, true)) {
+    putenv('APP_ENV=' . $_ENV['APP_ENV']);
+    // force loading .env files when --env is defined
+    $_SERVER['APP_ENV'] = null;
```
Comment (danrot) [inline bin/console.php]: This looks different from the recipe, right? Why is that?

File: bin/console.php [hunk 2]
```diff
@@ -4,35 +4,35 @@
...
 
+if ($_SERVER['APP_DEBUG']) {
+    umask(0000);
     if (class_exists(Debug::class)) {
         Debug::enable();
     }
 }
 
-$kernel = new Kernel($env, $debug, $suluContext);
+$kernel = new Kernel($_SERVER['APP_ENV'], $_SERVER['APP_DEBUG'], $suluContext);
```
Comment (danrot) [inline bin/console.php]: In the recipe `APP_DEBUG` is casted to a bool, why don't we do that?

**Review by danrot — CHANGES_REQUESTED** [review]

**Comment by alexander-schranz** [comment]
@danrot seems like I did forget some changes in the console.php. Did update it!

**Comment by danrot** [comment]
The comments in the current `.env` files are not correct anymore... It still refers to the `.env` file, and I think the content is also not true anymore (that's not a file that should be copied anymore, right?

**Review by danrot — APPROVED** [review]

**Pull request merged**